apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -92,12 +110,15 @@ export function InstructorAssessmentSettings({
                 <div class="form-group">
                   <label for="number">Number</label>
                   <input
-                    type="text"
+                    type="number"
```

Comment:
This is not true:

https://github.com/PrairieLearn/PrairieLearn/blob/18d8c068756bc4ac511281cd8443a55ec018e544/apps/prairielearn/src/schemas/schemas/infoAssessment.json#L30-L33

https://github.com/PrairieLearn/PrairieLearn/blob/18d8c068756bc4ac511281cd8443a55ec018e544/database/tables/assessments.pg#L26

Reply:
Reverted this to `type="text"` in [1c26d29](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/1c26d292a137fd79155de37b33bbdacb2b1d9127)